Protocol.md

```diff
@@ -273,6 +273,8 @@ rules MAY be used for handling layout. For example, `application/x.netflix+cbor`
 * __Setup Data__: includes payload describing connection capabilities of the endpoint sending the
 Setup header.
 
+__NOTE__: A server that receives a SETUP frame that has (__R__)esume Enabled set, but does not support resuming operation, must reject the SETUP with an ERROR. 
```

Comment:
I am not totally happy with this either. In the normal flow, there is no positive ack of SETUP success. Which means that changing things for RESUME_ENABLED is a bit odd and unclean.

Another option is to let everything be silent and allow non-resumable servers to continue. But this means a nasty surprise for the client if/when it tries to resume.

The cleanest solution I can think of is to require the non-resuming servers to error back. Yes, it is against the requirement of preventing required operation. But it is the cleanest and lowest cost way to get good behavior, IMO.